Add Quick bar option to Customizations reload (#21950)
home-assistant/home-assistant.github.io: source/_docs/configuration/customizing-devices.markdown

To reload customizations, navigate to Configuration > Server Controls and then press the "Reload Location & Customizations" button. If you don't see this, enable Advanced Mode on your user profile page first.

> You can alse use the [Quick bar](https://www.home-assistant.io/docs/tools/quick-bar/#hotkeys), and press "Reload Location & Customizations". If you don't see this, enable Keyboard shortcuts on your user profile page first.

Reviewer: Shortcuts are enabled by default, that note should not be repeated everywhere.

```suggestion
You can also use the [Quick bar](/docs/tools/quick-bar/#hotkeys), and choose "Reload Location & Customizations".
```
